src/core/plugins/core-items/server/src/updateItems.ts

```diff
@@ -0,0 +1,30 @@
+import alt from 'alt-server';
+import { Athena } from "../../../../server/api/athena";
+import { drinks } from './items/drinks';
+import { food } from './items/food';
+import { utility } from './items/utility';
+
+export class UpdateItems {
+    static async init() {
+        const itemsToUpdate = [
+            ...drinks,
+            ...food,
+            ...utility
```

Comment:
same here - you have 2 places to update your imports / items when adding another file - e.g. weapon items